README.md

```diff
@@ -1 +1,4 @@
 # movie-service
+
+
+A collecition of all movies
```

Comment:
collection instead of coll

Reply:
https://github.com/lavanyakottapalli/movie-service/pull/1/commits/a76afc19c05a95d544871ab4c49c4459ef38033f